packages/shared/src/components/post/MarkdownPostContent.tsx

```diff
@@ -16,12 +17,16 @@ function WelcomePostContent({ post }: WelcomePostContentProps): ReactElement {
         {post.title}
       </h1>
       {post.type === PostType.Freeform && post.image && (
-        <Image
-          src={post.image}
-          alt="Post cover image"
-          className="object-cover mb-10 w-full h-auto rounded-xl max-h-[18rem]"
-          fallbackSrc={cloudinary.post.imageCoverPlaceholder}
-        />
+        <Link href={post.image}>
+          <a target="_blank">
```

Comment:
Well need rel attributes here to be safe

Reply:
`rel="noopener"`?

Reply:
I personally like `noopener noreferrer` but that alone will do 👌

Reply:
Added `noreferrer` as well.

Reply:
Thanks should be good to merge